### ISSUE TITLE
Accessibility: Button Color Contrast

### ISSUE THREAD

**Issue by chrisdemars** [vuejs/vuepress#273]
The color contrast on the "Get Started" button on the home page does not pass WCAG 2.0 AA standards. I would like to fix this.

**Comment by ulivz** [comment]
Welcome, but, can you explain to me what is `WCAG 2.0 AA standards`. 😅

**Comment by chrisdemars** [comment]
@ulivz WCAG 2.0 standards are the Web Content Accessibility Guidelines v2.0. There are three levels of conformance, A, AA, AAA. Level A should always be hit, you want to strive for level AA. There is no need to worry about AAA because it is very difficult to hit that level of conformance. The color contrast of the background color and foreground color of the button does not meet proper color ratios. My code would fix that and clear up one of the violations.

**Comment by chrisdemars** [comment]
Ok, I will submit more accessibility issues.
